src/Components/Molecules/grid-image/grid.js

```diff
@@ -0,0 +1,16 @@
+import './grid.scss';
+
+const Grid = (images) => {
+    return (
+        <div className="about__image-grid">
+            <div className="about__back-container">
+                <img src={images.top} alt={images.altTagTop}/>
+            </div>
+            <div className="about__front-container">
```

Comment:
Do you actually need to wrap these img in div?
Why not apply styling directly to img?

Reply:
Didn't thought about that. Made changes now in grid.js and also in grid.scss file.